Bottom: f70074932eb6324590761620a53b9137a39ff08e
Top:    92172e6ae3e3dc25e86139a1074cb5551a2c27db
Author: Ian Kent <raven@themaw.net>
Date:   2013-07-30 17:31:27 +0800

Refresh of autofs-5.0.7-add-dumpmap-to-file.patch

---

diff --git a/daemon/automount.c b/daemon/automount.c
index 4aaeac3..1970edc 100644
--- a/daemon/automount.c
+++ b/daemon/automount.c
@@ -2031,7 +2031,7 @@ int main(int argc, char *argv[])
 
 		case 'm':
 			dumpmaps = 1;
-			if (optstring)
+			if (optarg)
 				dump = getstropt(optarg, opt);
 			break;
 
@@ -2178,7 +2178,7 @@ int main(int argc, char *argv[])
 		lookup_nss_read_master(master_list, 0);
 		master_show_mounts(master_list, dump);
 		if (dump)
-			free(dump);
+			free((void *) dump);
 		exit(0);
 	}
